README.md

```diff
@@ -1,31 +1,21 @@
-# Conduit Connector for <resource>
-[Conduit](https://conduit.io) for <resource>.
+# Conduit Connector for Notion
+A [Conduit](https://conduit.io) connector for [Notion](https://www.notion.so).
 
 ## How to build?
 Run `make build` to build the connector.
 
 ## Testing
 Run `make test` to run all the unit tests. Run `make test-integration` to run the integration tests.
 
-The Docker compose file at `test/docker-compose.yml` can be used to run the required resource locally.
-
 ## Source
-A source connector pulls data from an external resource and pushes it to downstream resources via Conduit.
-
-### Configuration
-
-| name                  | description                           | required | default value |
-|-----------------------|---------------------------------------|----------|---------------|
-| `source_config_param` | Description of `source_config_param`. | true     | 1000          |
-
-## Destination
-A destination connector pushes data from upstream resources to an external resource via Conduit.
+TBD
```

Comment:
This is source-only at the moment right? If so, may be worth calling that out

Reply:
Good suggestion, I'll do that.